app/artifact-cas/internal/conf/conf.proto

```diff
@@ -88,4 +89,9 @@ message Credentials {
       // mount path of the kv engine, default /secret
       string mount_path = 3;
     }
+
+    message GCPSecretManager {
```

Comment:
It would be useful to add an example of this in the config.yaml.sample files i.e `artifact-cas/configs/samples/config.yaml`

In there you could add smth like

```yaml
# gcp_secret_manager:
#    project_id: my-project
#    auth_key: ...
```